README.md

```diff
@@ -115,14 +87,41 @@ To test the working of exporter, deploy test deployment that request resources
 make deploy-pod
 ```
 
-NOTE
-This exporter assumes the cluster has devices configured using [Sample device plugin](https://github.com/swatisehgal/sample-device-plugin)
-
 ## Limitations
 
-Due to the current (2020, July) limitations of both CRI and podresources APIs for the purposes of the resource-topology-exporter,
-we have explicit support for most commonly used device plugins.
+* RTE assumes the devices are not created dynamically.
```

Comment:
for the future: we may want to crosscheck and hint that AFAIK _neither_ kubelet (1.20) handles hotplug, so this limitation is actually dwarfed by kube's limitation.